apps/demos/menuMeta.json

```diff
@@ -3313,6 +3313,24 @@
       }
     ]
   },
+  {
+    "Name": "Chat",
+    "Equivalents": "dxChat, dx-chat, chat",
+    "Groups": [
```

Comment:
I think we need no group here and can just specify demos directly how it's done for DateRangeBox for example

Reply:
2679e0c3362d769e1ea3315578e64523e07689d8

Reply:
Actually, all categories at the root level have groups. The Chat should be at the root level, just like it is on the site right now

<img width="313" alt="image" src="https://github.com/user-attachments/assets/585fcaa9-0115-4614-b890-bba78058abab">

I suppose the first variant is more correct. The DateRangeBox is already part of the group.

@Strider2342 kindly revert to the previous variant with groups and let's add "DocUrl": "", as it's done for the neighboring demos

Reply:
ddf93820f6c2dcce14af6cf503d0775eea997f04